src/common/helpers/poller.js

```diff
@@ -0,0 +1,45 @@
+import logging from '../../server/logging';
+
+import { getGitHubRepoUrl } from './github-url';
+import { hasRepoChanged } from '../../server/handlers/github';
+import { internalGetSnapcraftYaml } from '../../server/handlers/launchpad';
+
+
+const logger = logging.getLogger('poller');
+
+
+// Extracts unique repository URLs from a given (parsed) snapcraft.yaml
+export function extractPartsToPoll(snapcraft_yaml) {
+  const parts = Object.values(snapcraft_yaml.parts || {});
+  const gh_parts = parts.filter(function (p) {
+    return ((p.source || '').startsWith('https://github.com/') &&
```

Comment:
Correct. Let's drop the '.git' clause for now and go only with `startsWith('https://github.com/')`. We can have a better idea of the currently registered repos when it reaches QA.